Scalar.Common/Git/GitVersion.cs

```diff
@@ -43,6 +45,11 @@ public GitFeatureFlags GetFeatures()
                 flags |= GitFeatureFlags.MaintenanceBuiltin;
             }
 
+            if (this.Extra != null && this.Extra.Equals("exp"))
```

Comment:
There was the idea of teaching `git version --build-options` to output that this version supports certain features, e.g. in a line `supports: fsmonitor` or some such. I think we'll need to coordinate this between you and @jeffhostetler.

Reply:
That is certainly an idea that we can pursue. I'm just following the existing patterns that Scalar already uses.

Reply:
Sorry, I was wrong: the line reads "feature: fsmonitor--daemon". See https://github.com/jeffhostetler/git/commit/998a08f7467561421ca5222fca80d74f249e6379#diff-c1fcc4ee5ac7cbf3ae60e007aa4db89a135cc30e939eb42cb8fb8502a7c7b472R670

Reply:
See https://github.com/github/git-client/issues/252

Reply:
>There was the idea of teaching git version --build-options to output that this version supports certain features, e.g. in a line supports: fsmonitor or some such. I think we'll need to coordinate this between you and @jeffhostetler.

This method sounds like a much more robust way to detect fsmonitor IMO.